neuros/board/board_interface.py

```diff
@@ -0,0 +1,32 @@
+# neuros.board.board_interface.py
+
+from brainflow.board_shim import BoardShim, BrainFlowInputParams, BoardIds
+
+
+class BoardInterface:
```

Comment:
Why have a BoardInterface that's different from the BoardShim already provided by the library?

Reply:
Upon reflection, the only real benefit is the individual channel configuration abstraction (not shown since I rewound and am doing smaller PRs). The default board only lets us access with raw indices, rather than by a label. I also just wasn't sure if we would hit board-specific annoyances when we switch to a real board. And I was going to ask later if you wanted reading and writing (there's a separate board for that).

But none of that is functionally necessary. So *shrugs*